doc/glossary.rst

```diff
@@ -100,98 +80,91 @@ Glossary of Terms
+      is started.  Such a User VM runs independently of and is partitioned from
+      the Service VM and other post-launched VMs. It has its own carefully
+      configured and dedicated hardware resources such as CPUs, memory, and I/O
+      devices. Other VMs, including the Service VM, may not even be aware of a
+      pre-launched VM's existence. A pre-launched VM can be used as a
+      special-case :term:`Safety VM` for reacting to critical system failures.
+      It cannot take advantage of the Service VM or Device Model services.
 
    Post-launched VM
-      Post-Launched VMs are launched and configured by the Service VM.
-      (See :term:`Pre-launched VM`)
-
-   PTE
-      Page Table Entry
-
-   PV
-      Para-virtualization (See
-      https://en.wikipedia.org/wiki/Paravirtualization)
-
-   PVINFO
-      Para-Virtualization Information Page, a MMIO range used to
-      implement para-virtualization
+      A User VM configured and launched by the Service VM and typically
+      accessing shared hardware resources managed by the Service VM and Device
+      Model. Most User VMs are post-launched while special-purpose User VMs are
+      pre-launched.
 
    QEMU
       Quick EMUlator.  Machine emulator running in user space.
 
-   RSE
-      Rear Seat Entertainment
-
    RDT
       Intel Resource Director Technology (Intel RDT) provides a set of
       monitoring and allocation capabilities to control resources such as
       Cache and Memory. ACRN supports Cache Allocation Technology (CAT) and
       Memory Bandwidth Allocation (MBA).
 
    RTVM
-      Real-time VM. A specially-designed VM that can run hard real-time or
-      soft real-time workloads (or applications) much more efficiently
-      than the typical User VM through the use of a passthrough interrupt
-      controller, polling-mode Virtio, Intel RDT allocation features (CAT,
-      MBA), and I/O prioritization.  RTVMs are typically a :term:`Pre-launched VM`.
-      A non-:term:`Safety VM` with real-time requirements is a
-      :term:`Post-launched VM`.
+   Real-time VM
+      A :term:`User VM` configured specifically for real-time applications and
+      their performance needs. ACRN supports near bare-metal performance for a
+      post-launched real-time VM by configuring certain key technologies or
+      enabling device-passthrough to avoid common virtualization and
+      device-access overhead issues. Such technologies include: using a
+      passthrough interrupt controller, polling-mode Virtio, Intel RDT
+      allocation features (CAT, MBA), and I/O prioritization.  RTVMs are
+      typically a :term:`Pre-launched VM`.  A non-:term:`Safety VM` with
+      real-time requirements is a :term:`Post-launched VM`.
 
    Safety VM
       A special VM with dedicated hardware resources, running in
       partition mode, and providing overall system health-monitoring
-      functionality.  Currently, a Safety VM is always a pre-launched User VM.
```

Comment:
Consider removing "running in partition mode" since "What Is ACRN" page shows Safety VM can exist in a hybrid scenario too.